packages/utils/tool-utils/.eslintrc.js

```diff
@@ -7,6 +7,9 @@ module.exports = {
     "extends": [
         "@fluidframework/eslint-config-fluid/eslint7"
     ],
+    "parserOptions": {
+        "project": ["./tsconfig.json", "./src/test/tsconfig.json"]
+    },
```

Comment:
@curtisman I changed this package as per the changes in #4626. Can you please take a look if this looks right.
For example, I added a few build scripts to package.json below which I thought was missing in this package.
Also, updated the tsconfigs since I added tests to the package.